test/testhelpers.sh

```diff
@@ -70,7 +70,8 @@ refute_server_object() {
   curl -v "$GITSERVER/$reponame.git/info/lfs/objects/$oid" \
     -u "user:pass" \
     -o http.json \
-    -H "Accept: application/vnd.git-lfs+json" 2>&1 |
+    -H "Accept: application/vnd.git-lfs+json" \
+    -H "X-Ignore-Retries: true" 2>&1 |
```

Comment:
`X-Ignore-Retries` is added as a header value here so as to not fail these `assert_server_object` when the objects that are being looked for have a failure count assosciated with them.